aws-secrets

```diff
@@ -0,0 +1 @@
+AWS_SECRET_ACCESS_KEY = wJalrXUtnFEMI/K7MDENG/bPxRfiCYSECRETSKEY
```

Comment:
Copied from the same file on aws-secret branch
https://github.com/secretlint/secretlint-github-actions-example/blob/1729c7d6f3340734a1936b4d98192c04fba8db31/aws-secrets